lib/tty/reader/key_event.rb

```diff
@@ -26,7 +26,7 @@ class KeyEvent < Struct.new(:key, :value, :line)
       # @return [KeyEvent]
       #
       # @api public
-      def self.from(keys, char, line = "")
+      def self.from(keys, char, line = nil)
```

Comment:
I believe the equivalent of an empty string is a `Line` object with an empty content.